lib/project_types/theme/commands/common/root_helper.rb

```diff
@@ -35,6 +37,23 @@ def root_value(options, name)
           "."
         end
 
+        def theme_directory?(root)
+          Theme::Models::SpecificationHandlers::Theme.new(root).valid?
+        end
+
+        def current_directory_confirmed?
+          Forms::ConfirmStore.ask(
+            @ctx,
+            [],
+            title: @ctx.message("theme.confirm_current_directory"),
+            force: options.flags[:force],
```

Comment:
The two existing `ConfirmStore.ask`s support the `--force` flag.
As it also makes sense for this PR's confirmation dialogue, I added it to all commands.

Reply:
we should definitely get input from UX before making an overarching change like this 😌 

functionally i think this makes sense, we're giving developers the previous behavior where you can run theme commands in any directory and allowing them to do that with `--force`

also after poking around in the repo a bit, i don't see any classes that try to directly look at this flag. so unexpected downstream effects seem unlikely 😄

Reply:
alternatively maybe a new flag make sense? `--ignore-theme-validation` or something of that kind? `--force` comes off as pretty destructive of a flag (cc: git push --force lol)

Reply:
> we should definitely get input from UX before making an overarching change like this 😌

Agreed. I will wait for Guilherme's input, before asking Meredith then :)

> also after poking around in the repo a bit, i don't see any classes that try to directly look at this flag. so unexpected downstream effects seem unlikely 😄

Thanks for checking too :)

> maybe a new flag make sense? --ignore-theme-validation or something of that kind? --force comes off as pretty destructive of a flag (cc: git push --force lol)

Contouring the validation with `--force` _is_ kinda destructive: It can overwrite a potentially valid theme with some irrelevant/invalid files 😄.